src/bundle.c

```diff
@@ -723,6 +727,53 @@ static int install_bundles(struct list *bundles, struct list **subs, int current
 	to_install_files = filter_out_existing_files(to_install_files);
 	grabtime_stop(&times);
 
+	/* Step 1.5: Now that we know the bundle is valid, check if we have enough space */
```

Comment:
future: the content of this `if` block should be refactored into its own function so we don't grow `install_bundles` to much. This function can be re-used when doing a `verify --install` then too.